keras_contrib/applications/densenet.py

```diff
@@ -164,6 +166,114 @@ def DenseNet(depth=40, nb_dense_block=3, growth_rate=12, nb_filter=16,
     return model
 
 
+def DenseNetFCN(nb_dense_block=5, growth_rate=12, nb_filter=16, nb_layers_per_block=4,
```

Comment:
Any original paper can of course be improved upon. :-) Specifying the number of dense block layers separately from the number of dense blocks is simply a better parameterization. It is also possible to support the original parameterization and the newer one.